### ISSUE TITLE
Infer other addresses from SystemConfig once MCP L1 is launched

### ISSUE THREAD

**Issue by zchn** [ethereum-optimism/superchain-registry#71]
The goal of this file to serve as a reference and source of truth, which we verify in the `CheckSecurityConfigs` script? I ask because as of https://github.com/ethereum-optimism/optimism/pull/6555, we can now only define a chain's `SystemConfig` and then discover everything else. Wondering if we can leverage that in this repo (once everyone is on MCP L1 contracts)

_Originally posted by @mds1 in https://github.com/ethereum-optimism/superchain-registry/pull/69#discussion_r1471464207_

**Comment by geoknee** [comment]
The downside of this is that the other addresses will no longer be tracked in the repository, and therefore not inspectable by humans / services / software packages that want to read them.

**Comment by mds1** [comment]
@geoknee What do you mean exactly? We can, for example, only ask for their SystemConfig address if it simplifies the add chain process, but still generate output files that contain all other system addresses for humans / services / software packages that want to read them.

**Comment by geoknee** [comment]
Yeah you're right that should work well. I think I was skipping all the way to "only store the SystemConfig address" which is not what this ticket is about.
